packages/drivers/routerlicious-driver/src/shreddedSummaryDocumentStorageService.ts

```diff
@@ -70,18 +84,40 @@ export class ShreddedSummaryDocumentStorageService implements IDocumentStorageSe
             this.blobCache = blobCache ?? new InMemoryCache();
             this.snapshotTreeCache = (snapshotTreeCache ?? new InMemoryCache()) as ICache<ISnapshotTreeEx>;
         }
+
+        this.getVersionsAggregateEvent =
```

Comment:
why bother spreading the object, and not just make aggregatePerformanceEvents private readonly

Reply:
if you simplify config, which i think you should, then you can just set threashold be be 1 if disableTelemetryAggregation

Reply:
Separating it into individual private members was a personal preference. Will change it to just being 1 private member